src/account/types/balance.rs

```diff
@@ -119,6 +119,18 @@ impl RequiredStorageDeposit {
     pub fn new() -> RequiredStorageDeposit {
         RequiredStorageDeposit::default()
     }
+    pub fn alias(&self) -> u64 {
```

Comment:
Please also add there new functions to the changelog

Reply:
Should I add them to the rc.4 changes or should I to a new entry for rc.5?